From e465ace14b3af2b2d8e4d6fbfa8da3ef58df274d Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Fri, 1 Dec 2017 12:14:48 -0500
Subject: [PATCH 2/2] Don't set LD_LIBRARY_PATH in environment cache.

---
 python/catkin/environment_cache.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/python/catkin/environment_cache.py b/python/catkin/environment_cache.py
index 3defa52..dd5e353 100644
--- a/python/catkin/environment_cache.py
+++ b/python/catkin/environment_cache.py
@@ -112,6 +112,7 @@ def _append_comment(code, value):
 
 def _set_variable(code, key, value):
     if _is_not_windows():
-        code.append('export %s="%s"' % (key, value))
+        if key != 'LD_LIBRARY_PATH':
+            code.append('export %s="%s"' % (key, value))
     else:
         code.append('set %s=%s' % (key, value))
-- 
2.15.1

